.palantir/revapi.yml

```diff
@@ -249,3 +249,8 @@ acceptedBreaks:
       new: "method <T> T com.palantir.dialogue.Clients::callBlocking(com.palantir.dialogue.EndpointChannel,\
         \ com.palantir.dialogue.Request, com.palantir.dialogue.Deserializer<T>)"
       justification: "New API for blocking calls."
+  "2.10.0":
```

Comment:
Technically if someone published a library that implemented Response, and then someone else pushed used that, they'd get a runtime error.

Reply:
Yep, that's fine. We should mention it in the changelog, but I don't expect any external Response implementations. We could call this a `break` and major rev because numbers are cheap.

Reply:
alright calling this a break